tests/features/collection/menu.feature

```diff
@@ -1,29 +1,55 @@
+# If you want to run this test through Selenium, uncomment the specified lines!
 @api
 Feature: Collections menu
   Scenario: Add a link to og menu
     Given collections:
-      | name              | description            | owner   | uri                                 |
-      | MenuCollection1   | First menu collection  |         | http://joinup.eu/collection/ogmenu1 |
-      | MenuCollection2   | Second menu collection |         | http://joinup.eu/collection/ogmenu2 |
+      | name               | description                     | owner | uri                                  |
+      | Lord of the rings  | A collection for true LOTR fans |       | http://joinup.eu/collection/lotr     |
+      | Star Trek          | A collection for Trekkies       |       | http://joinup.eu/collection/startrek |
     And users:
       | name      | roles      |
-      | Mr Menu   | moderator  |
-    # Create a link
-    Given I am logged in as "Mr Menu"
-    When I go to the homepage of the "MenuCollection1" collection
+      | Gandalf   | moderator  |
+    # No links in new menu
+    Given I am logged in as "Gandalf"
+    When I go to the homepage of the "Lord of the rings" collection
     Then I should see the link "Add menu"
     When I click "Add menu"
     Then I should see "There are no menu links yet."
+    # Add first link.
     When I click "Add link"
-    And I fill in "Menu link title" with "My user account page for the collection menu"
+    And I fill in "Menu link title" with "Mines of Moria"
+    And I fill in "Link" with "/user"
+    Then I press the "Save" button
+    Then I should see the success message "The menu link has been saved."
+    # Add second link.
+    When I click "Add link"
+    And I fill in "Menu link title" with "Mordor"
     And I fill in "Link" with "/user"
     Then I press the "Save" button
     Then I should see the success message "The menu link has been saved."
 
     # Check for the link on the collection
-    When I go to the homepage of the "MenuCollection1" collection
-    Then I should see the link "My user account page for the collection menu"
+    When I go to the homepage of the "Lord of the rings" collection
+    # Check order of links.
+    Then I should see the following in the repeated ".menu-item" element within the context of the ".block-og-menu ul.menu .menu-item" element:
+      | text           |
+      | Mines of Moria |
+      | Mordor         |
+
+    # Edit menu and change the order of items.
+    Then I click the contextual link "Edit menu" in the "Primary menu" region
```

Comment:
Since the contextual link is hidden behind an icon, we should include a comment here for the user to be able to know how to find it.